src/SkipTracebackWidget.ts

```diff
@@ -53,7 +62,7 @@ export default class SkipTracebackWidget
         this.node.appendChild(this._tracebackNode);
       } else {
         this._toggleBtn.className = TOGGLE_CLOSED_CLASS;
-        this._shortError.innerHTML = messageInCollapsedState(this._data);
+        addMessageInCollapsedState(this._shortError, this._data);
```

Comment:
nitpick, this is more of a setter so can we name this `setMessageInCollapsedState`?